src/main/java/com/schedule/workout/workoutScheduler/services/RolesService.java

```diff
@@ -0,0 +1,88 @@
+package com.schedule.workout.workoutScheduler.services;
+
+import com.schedule.workout.workoutScheduler.controller.role.CreateRoleModel;
+import com.schedule.workout.workoutScheduler.controller.role.RoleModel;
+import com.schedule.workout.workoutScheduler.controller.role.UpdateRoleModel;
+import com.schedule.workout.workoutScheduler.database.IRolesRepository;
+import com.schedule.workout.workoutScheduler.database.model.RoleDB;
+import com.schedule.workout.workoutScheduler.exceptions.RoleAlreadyExists;
+import com.schedule.workout.workoutScheduler.exceptions.RoleNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class RolesService {
+
+    @Autowired
+    IRolesRepository rolesRepository;
+
+
+    public CreateRoleModel createRole(CreateRoleModel createRoleModel) {
+        List<RoleDB> roles = rolesRepository.findByName(createRoleModel.getName());
```

Comment:
Role should be unique per name, so this should either return null or single role object.